deepchecks/vision/checks/train_test_validation/new_labels.py

```diff
@@ -118,10 +109,10 @@ def compute(self, context: Context) -> CheckResult:
             for class_id, num_occurrences in classes_only_in_test_count.items():
                 # Create id of alphabetic characters
                 sid = ''.join([choice(string.ascii_uppercase) for _ in range(3)])
-                images_of_class_id = \
-                    list(set(test_data.classes_indices[class_id]))[:self.max_images_to_display_per_label]
-                images_combine = ''.join([f'<div class="{sid}-item">{draw_image(test_data, x, class_id)}</div>'
-                                          for x in images_of_class_id])
+                thumbnail_images = [draw_image(img, labels) for img, labels in
```

Comment:
What happens here if not .has_image?

Reply:
same as now, we show an 'Image unavailble' where the image should be placed